docs/Advanced Features/Working with Manifests.md

```diff
@@ -2,20 +2,31 @@ Every data package is backed by a **manifest**. A manifest is a self-contained r
 
 Every time you save a data package to a registry you also save its manifest. You can inspect the manifest yourself using the `manifest` property:
 
-```bash
-$ python
->>> import quilt3
->>> p = quilt3.Package().set("foo.txt", "foo.txt")
->>> list(p.manifest)
-<<< [
-        {'version': 'v0'},
-        {'logical_key': 'Roadmap.md',
-         'physical_keys': ['file:///.../foo.txt'],
-         'size': 1000,
-         'hash': None,
-         'meta': {}
-        }
-    ]
+
+
+```python
+import quilt3
+import os
+
+# create test file
+f = open("foo.txt", "w")
+f.write("test content...")
+f.close()
+
+p = quilt3.Package().set("foo.txt", "foo.txt")
+list(p.manifest)
+```
+
+```shell
```

Comment:
this is a case where we want to leave outputs on. The way we control what prints an output and what doesn't should be via code sample rather than nbconvert.